src/main/frontend/components/block.cljs

```diff
@@ -245,7 +245,10 @@
                                       images (if-not (= (count images) 1)
                                                (let [^js _image (.closest (.-target e) ".asset-container")
                                                      image (. _image querySelector "img")]
-                                                 (cons image (remove #(= image %) images)))
+                                                 (->> images
+                                                      (sort-by (juxt #(.-y %) #(.-x %)))
+                                                      (split-with (complement #{image}))
+                                                      reverse (apply concat)))
```

Comment:
Better add a newline after `reverse` according to Clojure style guide. https://github.com/bbatsov/clojure-style-guide#threading-macros
```suggestion
                                                      reverse
                                                      (apply concat)))
```